deployer.sh

```diff
@@ -44,22 +42,6 @@ while [[ $1 != "" ]] ; do
             shift
             ;;
 
-        "--pull-all"|"-a")
```

Comment:
why did you remove those options? were they not useful?

Reply:
It's not used anywhere. I removed all params that are not used yet. If we use it we can put it in again.